internal/controller/ca_bundle.go

```
@@ -0,0 +1,185 @@
+		return fmt.Errorf("certificate data is nil")
+	}
+
+	rest := pemData
+	for {
+		var block *pem.Block
+		block, rest = pem.Decode(rest)
+		if block == nil {
+			break
+		}
+
+		if block.Type != "CERTIFICATE" {
```

golangci-lint (2.12.2)

[error] string `CERTIFICATE` has 5 occurrences, make it a constant

(goconst)